### PR TITLE
Add Travis CI support

### PR DESCRIPTION
**Pull request by marcegl91**
### Summary

- Implemented  TravisCI support.

### Trello Card

https://trello.com/c/m0VFUeZ7/12-configurar-travisci

### CODE REVIEW

File: wbooks-api/Gemfile [hunk 1]
```diff
@@ -39,6 +39,8 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+  gem 'rspec-rails', '~> 3.8.2'
+  gem 'scss_lint', '~> 0.57.1'
```
Comment (facucompe) [inline wbooks-api/Gemfile:43]: Why did you add this gem?

File: wbooks-api/Gemfile [hunk 2]
```diff
@@ -48,6 +50,7 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rubocop'
```
Comment (facucompe) [inline wbooks-api/Gemfile]: Merge the changes that you did in rubocop branch

File: wbooks-api/Gemfile (hunk 1 above)
Comment (facucompe) [reply wbooks-api/Gemfile:43]: nothing said, my bad

**Review by floresv — APPROVED** [review]
LGTM
